Taxi/Taxi/Presenter/SignUpCodeView.swift

```diff
@@ -0,0 +1,104 @@
+//
+//  SignUpCodeView.swift
+//  Taxi
+//
+//  Created by sanghyo on 2022/06/14.
+//
+
+import SwiftUI
+
+struct SignUpCodeView: View {
+    @Environment(\.dismiss) private var dismiss
+    @State private var signUpCode: String = ""
+    @State private var codeState: FieldState = .normal
+    @State private var isActive: Bool = false
```

Comment:
```suggestion
    @State private var signUpCode = ""
    @State private var codeState: FieldState = .normal
    @State private var isActive = false
```

값에서 바로 예상가능하는 타입은 생락되어도 된답니다 ~~~ !